sdk/azcore/testdata/perf/go.mod

```diff
@@ -1,15 +1,17 @@
 module github.com/Azure/azure-sdk-for-go/sdk/azcore/testdata/perf
 
-go 1.18
+go 1.23.0
+
+toolchain go1.24.0
```

Comment:
If we want to run the perf tests against both versions in CI, then this should be removed.

Reply:
Thanks, I overlooked this one